src/lib/redux/siteData.ts

```diff
@@ -25,4 +25,8 @@ const setCurrentSite = (newSite: any): void => {
   store.dispatch(setCurrSite(newSite));
 };
 
-export { refreshSiteData, setCurrentSite };
+const addCustomer = (customer: any): void => {
```

Comment:
Is this the right place to have this? It doesn't feel super related to site (I guess it's just adding a customer to the current site?).